src/app/src/components/DashboardAdjustMatchCard.jsx

```diff
@@ -392,6 +392,12 @@ export default function DashboardAdjustMatchCard({
                                 label="Confidence Score"
                                 value={match.confidence}
                             />
+                            {typeof match.transferred_from === 'string' && (
```

Comment:
We don't use `typeof` elsewhere in the app and JavaScript types can be tricky. Since we are looking for a non-null value here, consider

```javascript
import isNil from 'lodash/isNil';
```

and 

```javascript
                            {!isNil(match.transferred_from) && (
```